plugin/fiftyone/app/main.py

```diff
@@ -0,0 +1,34 @@
+import logging
```

Comment:
check https://google.github.io/styleguide/pyguide.html#22-imports
for import orders.